documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -95,7 +95,13 @@ Capacity limits can be set for the following broker resources:
 For disk storage, use a number value with either standard Kubernetes byte units (K, M, G, or T), their bibyte (power of two) equivalents (Ki, Mi, Gi, or Ti), or a byte value with or without E notation.
 For network throughput, use an integer value with standard Kubernetes byte units (K, M, G) or their bibyte (power of two) equivalents (Ki, Mi, Gi) per second.
 
-Because Strimzi Kafka brokers are homogeneous, Cruise Control applies the same capacity limits to every broker it is monitoring.
+For Kafka clusters that do not use JBOD storage, Strimzi generates a generic Cruise Control capacity configuration entry that applies to every broker.
```

Comment:
???

```suggestion
For Kafka clusters that use only one disk per broker, Strimzi generates a generic Cruise Control capacity configuration entry that applies to every broker.
```

Reply:
For Kafka clusters that do not use JBOD storage, Strimzi will generate a capacity configuration like this:
```
{
  "brokerCapacities":[
    {
      "brokerId": "-1",
      "capacity": {
        "DISK": "100000",
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "This is the default capacity. Capacity unit used for disk is in MB, cpu is 
              in percentage, network throughput is in KB."
    }
  ]
}
```
Where the default broker entry `-1` is applied to every broker in the cluster.

For Kafka clusters that use JBOD storage, Strimzi will generate a capacity configuration like this:
```
{
  "brokerCapacities":[
    {
      "brokerId": "0",
      "capacity": {
        "DISK": {
          "/var/lib/kafka/data-0/kafka-log0" : "1024.0"
          "/var/lib/kafka/data-1/kafka-log0" : "1024.0"
        },
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "Capacity for Broker 0"
    },
    {
      "brokerId": "1",
      "capacity": {
        "DISK": {
          "/var/lib/kafka/data-0/kafka-log1" : "1024.0"
          "/var/lib/kafka/data-1/kafka-log1" : "1024.0"
        },
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "Capacity for Broker 1"
    },
    {
      "brokerId": "1",
      "capacity": {
        "DISK": {
          "/var/lib/kafka/data-0/kafka-log2" : "1024.0"
          "/var/lib/kafka/data-1/kafka-log2" : "1024.0"
        },
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "Capacity for Broker 2"
    }
  ]
}

```
Even if it is a JBOD configuration that only has one disk:

```
{
  "brokerCapacities":[
    {
      "brokerId": "0",
      "capacity": {
        "DISK": {
          "/var/lib/kafka/data-0/kafka-log0" : "1024.0"
        },
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "Capacity for Broker 0"
    },
    {
      "brokerId": "1",
      "capacity": {
        "DISK": {
          "/var/lib/kafka/data-0/kafka-log1" : "1024.0"
        },
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "Capacity for Broker 1"
    },
    {
      "brokerId": "1",
      "capacity": {
        "DISK": {
          "/var/lib/kafka/data-0/kafka-log2" : "1024.0"
        },
        "CPU": "100",
        "NW_IN": "10000",
        "NW_OUT": "10000"
      },
      "doc": "Capacity for Broker 2"
    }
  ]
}

```

Reply:
How should people know that this is what _generic_ means?

Reply:
I am not sure but I feel like I am getting users into the weeds with this, I am going to add some logic in the Capacity class to prevent users from misconfiguration. It will be much easier for us and the users to that then explain and have them understand!